CorsixTH/Lua/dialogs/fullscreen/fax.lua

```diff
@@ -282,7 +282,7 @@ function UIFax:close()
   self.icon.fax = nil
   self.icon:adjustToggle()
   UIFullscreen.close(self)
-  if world and world:isCurrentSpeed("Pause") then
+  if world and world:isCurrentSpeed("Pause") and #self.ui:getWindows(UIStaffRise) == 0 then
```

Comment:
I understand the point of this from our discussions in IRC but only from reading the code it's not clear why this is happening. Could we perhaps create a function to count all other pause dialogs that might be open to use in both cases?